managed/models/database.go

```diff
@@ -915,6 +915,12 @@ var databaseSchema = [][]string{
 		`ALTER TABLE agents
 		ALTER COLUMN comments_parsing_disabled DROP DEFAULT`,
 	},
+	85: {
+		`ALTER TABLE agents 
+		ADD COLUMN expose_exporter_address BOOLEAN NOT NULL DEFAULT FALSE;`,
```

Comment:
I believe the recommendation is not to use defaults, as per line 920. Let's see if we get away without it.

Reply:
we have to use default to set default value for `not null` columns, but we need to remove that once values are set.
So it should look like the migration above